packages/components/src/MultiSelect/MultiSelect.spec.tsx

```diff
@@ -281,3 +281,32 @@ describe("Removing an option", () => {
     })
   })
 })
+
+describe("Has validation status", () => {
+  it("has a validation message with an accessible name", () => {
+    render(
+      <MultiSelectWrapper
+        selectedValues={new Set(["waffle"])}
+        status="error"
+        validationMessage="No waffles are available"
+      />
+    )
+    expect(screen.getByText("No waffles are available")).toBeInTheDocument()
+  })
+  it("describes the toggle", () => {
```

Comment:
Definitely worth adding a test (or amending this one) to also showcase the order relative to the field description when Doug's PR is merged in.

Reply:
Added in a new test to testing the concatenation specifically in the accessible description 👍